pages/index.js

```diff
@@ -4,6 +4,8 @@ import { motion } from "framer-motion";
 import AppleLogo from "../components/AppleLogo";
 import FooterLink from "../components/FooterLink";
 import { stagger, slideIn } from "../lib/animations";
+import PlugLogo from '../public/images/logo.png'
+import PlugAppImage from '../public/images/app.png'
```

Comment:
Huh... Does the latest Next.js have Webpack image loaders?! 👀  🎉

Reply:
Yes indeed! https://nextjs.org/docs/api-reference/next/image